### PR TITLE
Remove media class from Media item wrapper

### PR DESCRIPTION
**Pull request by genjohnson**
TLDR: This PR removes the `.media` class from Media items, preventing Bootstrap from applying styles unexpectedly.

The full explanation...

With our recent updates to Media, items are wrapped in a div with several classes depending on the image style and selected alignment, for example `media  media-element-container media-program_small_square media-wysiwyg-align-left`.

The `media` classes is used by Bootstrap, which adds `.media { margin-top: 15px; }` along with some other styles.

This is causing unexpected spacing in content and there's no simple work-around for editors to remove this extra margin.
screenshot 2017-06-14 10 55 56

Removing the `.media` class in `sitenow_media_media_wysiwyg_token_to_markup_alter()`, the images align as expected.
screenshot 2017-06-14 10 56 14

### CODE REVIEW

**Comment by joewhitsitt** [comment]
I am surprised this got into a community release given all the bootstrap usage out there.

Is removing this globally the right choice? If so, I think this looks good. Otherwise, (though daunting), I feel like this is a theme issue and should be addressed there if Media itself doesn't correct the issue.

**Comment by richardbporter** [comment]
I was thinking the same thing. Does removing this impact anything else?

Otherwise, I tested locally and it seems to fix the alignment issue.

**Comment by genjohnson** [comment]
When I grepped through the themes on 703 this morning, only ovpred_compliance included the `.media` class, and it looks to be the styles provided by Bootstrap.

https://github.com/ITS-UofIowa/sitenow_custom_profile/blob/7.x-3.x/themes/ovpred_compliance/assets/css/ovpred_compliance.style.css

I also grepped through the Media module and it's not targeting the `.media` class. 

We could try opening an issue in Media to have the class removed there, but I don't expect it to far based on https://www.drupal.org/node/2194359.

**Comment by genjohnson** [comment]
I've opened [an issue on d.o about this](https://www.drupal.org/node/2886267), which also includes a patch if we want to switch methods for removing this class.
